### ISSUE TITLE
[Accessibility]A11y_VSCode_Editor_ScreenReader: Jaws does not read the list items of suggestions treeview in a meaningful sequence.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52441]
**Environment Details:** 
VSCode Version : 1.24.1
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.2
Jaws Version: 18.0.5038

**Repro Steps:**
1. Launch VS Code while Jaws is ON.
2. Open any file in the editor and type something.

**Actual:**
Jaws does not read the list items of suggestions treeview in a meaningful sequence. Suppose first list item has some details to it so after pressing down arrow key it reads the details and then focus goes to the background elements and we are not able to read the below list items.

**Expected:**
After reading the attached details, Jaws should read the next list item.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

Attachment for Reference: 
[A11y VSCode Editor Jaws focus order.pptx](https://github.com/Microsoft/vscode/files/2118862/A11y.VSCode.Editor.Jaws.focus.order.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
We do not support jaws
